src/ImportPage.css

```diff
@@ -14,11 +14,9 @@
 
 .link {
   color: #282c34;
-  transition: 0.2s;
```

Comment:
Keeping this in seems to fix the issue without removing the animations. Can you try and confirm?

Reply:
yeah I can try